### DISCUSSION TITLE
Deploy request failing

### DISCUSSION THREAD

**Discussion by skyf0l** [planetscale/discussion#628] [Bug Reports] [answered]
Hello, 

Since yesterday, I've been unable to deploy a branch in my production database (3 tries).
This deployment only adds two forein keys. How is this possible?
Hopefully, this merge is not mandatory for our business, but it would be nice to deploy it quickly.

Org: nema
Database: nema
Deploy request #56, #57 and #58 (pr-828 to deploy to stable)

![image](https://github.com/planetscale/discussion/assets/59019720/c4975e3f-4bad-4457-bfab-555edfb3c94b)

![image](https://github.com/planetscale/discussion/assets/59019720/3bd31ac4-4e93-4610-b7a2-b5e472c425c7)

**Comment by mscoutermarsh** [comment]
Hi @skyf0l, could you open a support ticket? We can best help you there. https://planetscale.com/contact

**Reply by skyf0l** [reply]
in progress

**Comment by skyf0l** [comment, marked as answer]
Ok, so after contacting support, it appears that there's a bug when foreign keys loop with an `ON DELETE SET NULL`. The safe migration prevented me from making this change in prod.

So I disabled safe migration and directly added my constraints to my production database (ouch). According to support, the problem should be solved within a few weeks.
